Automattic/wp-calypso
# Woo Express: Added custom labels to WX Plans CtA

* Added custom button labels to Woo Express Plans
* Added "Popular" tag for Performance plan

*This PR is not updating the features' copy. This is gonna be handled in a separate PR.
![image](https://user-images.githubusercontent.com/3801502/229528920-ef4ca273-3efc-405a-984a-9bd63c7241fe.png)


### Testing
* Open the Plans page using a trial site (`plans/site-slug`)
* Make sure the `plans/wooexpress-small` feature flag is enabled.
